lib/logstash/codecs/rubydebug.rb

```diff
@@ -15,6 +16,7 @@ class LogStash::Codecs::RubyDebug < LogStash::Codecs::Base
   AWESOME_OPTIONS = {}
 
   def register
+    logger.warn("This plugin is deprecated. It will be renamed to 'pretty' in the next major version")
```

Comment:
I suggest something like:

`This plugin is deprecated. Please use the "pretty" codec instead.`